code/datums/looping_sounds/looping_sound.dm

```diff
@@ -78,6 +80,11 @@
 /datum/looping_sound/proc/play(soundfile)
 	var/list/atoms_cache = output_atoms
 	var/sound/S = sound(soundfile)
+	if(area_sound)
+		for(var/area/sound_outputs in atoms_cache)
+			for(var/mob/listener in mobs_in_area(sound_outputs, TRUE))
+				S.volume = volume * (USER_VOLUME(listener, channel) || 1)
+				SEND_SOUND(listener, S)
 	if(direct)
```

Comment:
I'm worried this is gonna play the sound twice.

Any reason you've changed this as opposed to just the `channel = CHANNEL_ASH_STORM` change?

Reply:
just the channel change didn't work in testing, sound_outputs will return areas, so the sound will directly play from the area itself

Reply:
Ok but you dont `return` after this meaning it goes into the second loop below.

Reply:
I may be a _little_ stupid
Return applied, cut the "or 1" check at the end because it would result in people with ash storms fully off actually getting a loud storm, so you would need to set it to like 1

Reply:
Doesn't the latter also apply to the existing code then? If turned off fully, you get it as if it was set to 100%

Reply:
So the || 1 works again for some reason???
I'm convinced I'm just cursed at this point and can break things by looking at them too hard